### DISCUSSION TITLE
android transparent pointer

### DISCUSSION THREAD

**Discussion by TUPSI-art** [ppy/osu#18484] [Ideas] [answered]
Hello, I really don't like that the pointer is intentionally faded for 2 reasons, the first is that I don't feel like I have a good aim with the touch if it's not there and the second when I record my screen at the time i play the pointer is not visible, i would suggest that there be an option to hide it and show it, thank you very much

**Comment by bdach** [comment, marked as answer]
...so you bumped #14434 (which is the same exact issue) and then posted this? please don't do that, no point.
